lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -1202,11 +1320,14 @@ bool SessionState::is_active() {
 void SessionState::set_fsm_state(
     SessionFsmState new_state, SessionStateUpdateCriteria& uc) {
   // Only log and reflect change into update criteria if the state is new
+  uint32_t local_teid_ = get_upf_local_teid();
   if (curr_state_ != new_state) {
-    MLOG(MDEBUG) << "Session " << session_id_ << " Teid " << local_teid_
-                 << " FSM state change from "
-                 << session_fsm_state_to_str(curr_state_) << " to "
-                 << session_fsm_state_to_str(new_state);
+    MLOG(MINFO) << "Session id: " << session_id_ << " Teid: " << local_teid_
```

Comment:
This is code shared by lte/wlan. Are all of these values relevant?

Reply:
Chnaged as    MLOG(MDEBUG) << "Session id: " << session_id_ << " Teid: " << local_teid_
                 << " FSM state change from "
                 << session_fsm_state_to_str(curr_state_) << " to "
                 << session_fsm_state_to_str(new_state)
                 << " of imsi: " << get_subscriber_id()
                 << " 5G specific-PDU Id: " << get_pdu_id();

Reply:
This looks OK. Might better to only conditionally print the pdu id if the session is 5G.

Reply:
sure, I have taken care .